server/api/themoviedb/index.ts

```diff
@@ -364,13 +371,16 @@ class TheMovieDb extends ExternalAPI {
     sortBy = 'popularity.desc',
     page = 1,
     language = 'en',
+    region = '',
   }: DiscoverTvOptions = {}): Promise<TmdbSearchTvResponse> => {
     try {
       const data = await this.get<TmdbSearchTvResponse>('/discover/tv', {
         params: {
           sort_by: sortBy,
           page,
           language,
+          with_release_type: 6,
+          region: region,
```

Comment:
These are the correct parameters to filter `/discover/movie` by region, but doesn't seem to work for `/discover/tv`. I haven't seen anything in the API docs for it either. I am wondering if anyone has any idea?